### DISCUSSION TITLE
Version 3.0.0

### DISCUSSION THREAD

**Discussion by NeariX67** [KG-Development/SmartBMSUtility#30] [General]
We have finally released version 3.0.0, available now on the [Google Play Store](https://play.google.com/store/apps/details?id=com.nearix.smart_bms_utility&pcampaignid=web_share), iOS and MacOS Store, as well as the Windows Store.

Please use this discussion for feature requests and feedback, whereas Bugs are intended to be reported as an issue.

Cheers
Justin

**Comment by haekendes** [comment]
Hello there,

when trying to connect to my motorcycle's batteries, I must always find them by clicking three buttons; Add Device -> Unfiltered Search -> Activate.
If I don't, the batteries (and other devices) won't be found.

Could you add a config to set unfiltered search to always active?
I'd really appreciate if batteries that are set to auto connect would connect without having to press any buttons after opening the app, since I'm using your app every day.

Love using your app btw!

Cheers
Robin

**Reply by NeariX67** [reply]
Hello Robin,

thanks for your feedback. We will keep this in mind for future updates.

Kind regards
Justin
